src/components/Navigation/index.js

```diff
@@ -874,7 +882,7 @@ class Navigation extends Component {
     const pos = this.itemLoc(searchSelect);
     const label = as === 'pin' ? undefined : as;
     NavigationAPI.putLocationSave(dongleId, pos.lat, pos.lng,
-      this.formatSearchName(searchSelect), this.formatSearchDetails(searchSelect), 'favorite', label)
+      this.formatSearchName(searchSelect), this.formatSearchAddress(searchSelect), 'favorite', label)
```

Comment:
Need to make sure we are still giving the nav API the same information as before